src/components/NewWorkspaceModal.js

```diff
@@ -118,7 +118,8 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
     Ajax(signal).Groups.list().then(setAllGroups),
     !!cloneWorkspace && Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
       .then(locationResponse => {
-        setBucketLocation(locationResponse.location)
+        // For current phased regionality release, we only allow US workspace buckets.
+        isUSLocation(locationResponse.location) ? setBucketLocation(locationResponse.location) : setBucketLocation(defaultLocation)
```

Comment:
Could you nest the position of this ternary, and also destructure location?
```js
      .then(({ location }) => {
        setBucketLocation(isUSLocation(location) ? location : defaultLocation)
        setSourceWorkspaceLocation(location)
      })

```

Reply:
Done.